charts/seed-monitoring/charts/core/charts/prometheus/rules/kube-etcd3.rules.yaml

```diff
@@ -69,6 +69,51 @@ groups:
   - record: shoot:etcd_object_counts:sum_by_resource
     expr: sum(etcd_object_counts) by (resource)
 
+# etcd DB size alerts
+  - alert: KubeEtcd3MainDbSizeLimitApproaching
```

Comment:
We can combine the main and event alerts into one. This alert should be renamed to `KubeEtcd3DbSizeLimitApproaching`. The description will have to be changed to:
```yaml
description: Etcd3 {{ $labels.role }} DB size is approaching its current practical limit of 2GB.
```
and then we won't need multiple alerts for main and events. This should probably be done for the other ETCD alerts in a subsequent PR, but for right now let's focus on these alerts.